dev/com.ibm.ws.concurrent.persistent_fat_demo_timers/test-application/demotimers/src/ejb/timers/AutomaticDatabase.java

```
@@ -0,0 +1,132 @@
+    
+    /**
+     * Cancels timer execution
+     */
+    public void cancel() {
+        for (Timer timer : sessionContext.getTimerService().getTimers())
+            timer.cancel();
+    }
+
+    /**
+     * Get the value of count.
+     */
+    public int getRunCount() {
+        return count;
+    }
+    
+    /**
+     * Runs ever 30 seconds.  Automatically starts when application starts. 
+     */
+    @Schedule(info = "Performing Database Operations", hour = "*", minute = "*", second = "*/30")
+    public void run(Timer timer) {
+    	if(!isTableCreated)
+    		initTable();
+    	
+        System.out.println("Running execution " + incrementCount(timer) + " of timer " + timer.getInfo());
+    }
+    
+    private void initTable() {
+    	final String createTable = "CREATE TABLE AutomaticDatabase (name VARCHAR(64) NOT NULL PRIMARY KEY, count INT)";
+    	
+    	try (Connection conn = ds.getConnection()) {
+    		//See if table was created by another server
+        	DatabaseMetaData md = conn.getMetaData();
+        	ResultSet rs = md.getTables(null, null, "AutomaticDatabase", null);
+        	while(rs.next()) {
+        		isTableCreated = true;
+        		return;
+        	}
+    		
+    		//If not, create it.    		
+    		try (PreparedStatement pstmt= conn.prepareStatement(createTable)) {
+    			pstmt.executeUpdate();
+    		}
+    		isTableCreated = true;
+    	} catch (SQLException e) {
+			e.printStackTrace();
+		}
+    }
+    
+    private void initCounter(Timer timer) {
+    	final String createRow = "INSERT INTO AutomaticDatabase VALUES(?,?)";
+    	
+    	//create count
+    	count = 1;
+    	
+    	try (Connection conn = ds.getConnection()) {
+    		try (PreparedStatement pstmt = conn.prepareStatement(createRow)) {
+    			pstmt.setString(1, timer.getInfo().toString());
+    			pstmt.setInt(2, count);
+    			pstmt.execute();
+    		}
+    	} catch (SQLException e) {
+    		count = -1;
+			e.printStackTrace();
```

Should fail the test instead of suppressing the error, unless there is any reason why init logic is expected to fail and need a retry.